dist/modules/checkboxFilterGroup/checkboxFilterGroup.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var moduleName = 'c2fo.ui.modules.checkboxFilterGroup';
+
+angular.module(moduleName, [])
+    .directive('c2foCheckboxFilterGroup', [function () {
+        return {
+            restrict: 'E',
+            scope: {
+                label: '@',
+                fields: '=',
+                filter: '=',
+                filterKey: '@',
+                multicheck: '=',
+                onUpdate: '&'
+            },
+            template: [
+                '<div class="c2fo-ui-checkbox-filter-group">',
+                '   <div class="heading">{{checkBoxFilterGroupCtrl.label}}</div>',
+                '   <div class="checkbox" ng-repeat="field in checkBoxFilterGroupCtrl.fields">',
+                '       <label>',
+                '           <input type="checkbox" ng-model="field.isChecked" ng-click="checkBoxFilterGroupCtrl.updateFilter($event)"/>',
+                '           {{field.label}}',
+                '       </label>',
```

Comment:
This may be for the ui-guide later. 

We should document the reasons on why we want to wrap with a `<label>` and when we'd want it to be separated from the input.  In this case it makes sense so that you can have a "check" apply anywhere in the text or checkbox.